activiti-core/activiti-engine/src/test/java/org/activiti/engine/test/api/event/ActivityEventsTest.java

```diff
@@ -486,7 +485,7 @@ public void testActivityTimeOutEvent() {
     Calendar tomorrow = Calendar.getInstance();
     tomorrow.add(Calendar.DAY_OF_YEAR, 1);
     processEngineConfiguration.getClock().setCurrentTime(tomorrow.getTime());
-    waitForJobExecutorToProcessAllJobs(2000, 1000);
+    waitForJobExecutorToProcessAllJobs(5000, 1000);
```

Comment:
do you know why this timeout is impacted please?

Reply:
After JDK 17 update, this test was consistently timing out with 2 seconds...